packages/compiler/test/formatter/formatter.ts

```diff
@@ -27,12 +27,12 @@ describe("compiler: prettier formatter", () => {
       code: `
     import   "@cadl-lang/rest";
 import        "@azure-tools/cadl-autorest";
-import "@azure-tools/cadl-rpaas"  ;
+import "@azure-tools/cadl-providerhub"  ;
```

Comment:
not sure we should even have that in the public repo. Maybe the test can be updated with another import. Doesn't need to actually exists as its just for formatting

Reply:
Yes, this makes sense given the current public/private split. Though I think when layering came in, providerhub really gonna turn into ARM which is perfectly fine on public repo. For that matter, the only thing could potentially be on private may be providerhub controller related stuff. But even that can be disputed since ProviderHub is now public with public SDKs already. Let's talk about it in scrum.

Reply:
IMHO, it still makes sense to keep the core repo "core" and without references to microsoft-specific stuff is public.

Reply:
And even stronger than *references*, azure-specific components should never *live* in this repo.

Reply:
I don't think it's worth waiting until scrum (on Tuesday) since there is no reason for formatter tests to reference these libraries no matter what we decide in general case. I would go ahead and pick packages from "@cadl-lang" scope to replace all the `@azure-tools` references in this file.

Reply:
We could even use made up package names for this test.